lib/linguist/heuristics.rb

```diff
@@ -380,6 +380,26 @@ def call(data)
       end
     end
 
+    disambiguate ".pod" do |data|
+      if /^\s*=\w+$/.match(data)
+        if /^=pod|=cut/.match(data)
+          Language["Pod"]
+        elsif /^\s+=(begin|cut|pod)/.match(data)
+          Language["Pod 6"]
```

Comment:
I thought we had finally settled on using "Perl 6" as a language, as opposed to "Pod 6".

Reply:
Nope. I feel @samcv got the the crux of the “Pod 6” vs “Perl 6” debate with https://github.com/github/linguist/pull/3366#issuecomment-380282331 and commented as such in my follow up.

Reply:
I personally also agree that code and documentation are two separate things, and that we should handle them as such. Just because you can parse Markdown with Ruby doesn't make Markdown the same thing as Ruby.

Reply:
But Pod 6 is Perl 6, because it's interpreted by the Perl 6 interpreter itself, and parsed to Perl 6 structures. Markdown is not valid Ruby syntax, and needs an external module. But that's not the issue here, we're talking about what to call it in this context and I agree with @lildude and @samcv here.